File: Two Sum/twoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int, int> map;
        for (int i = 0; i < nums.size(); i++) {
            int complement = target - nums[i];
            auto found = map.find(complement);
            if (found != map.end()) {
                return {found->second, i};
            }
            map[nums[i]] = i;
        }
        
        return {0,0};
        
    }
};

File: Two Sum/twoSum.py
class Solution():
    def twoSum(self, nums, target):
        if len(nums) <= 1:
            return False
        map = {}
        for i in range(len(nums)):
            if nums[i] in map:
                return [map[nums[i]], i]
            else:
                complement = target - nums[i]
                map[complement] = i
